src/EPR.Accreditation.API/Controllers/SaveAndContinueController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using EPR.Accreditation.API.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using DTO = EPR.Accreditation.API.Common.Dtos;
+
+namespace EPR.Accreditation.API.Controllers
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+    public class SaveAndContinueController : ControllerBase
+    {
+        protected readonly IAccreditationService _accreditationService;
+
+        public SaveAndContinueController(IAccreditationService accreditationService)
+        {
+            _accreditationService = accreditationService ?? throw new ArgumentNullException(nameof(accreditationService));
+        }
+
+        [HttpGet("{id}")]
```

Comment:
We can save some writing here because ID is used in all the actions, we can move the parameter to line 8

Reply:
Amended